modyn/config/schema/modyn_config_schema.yaml

```diff
@@ -376,6 +376,25 @@ properties:
           The port on which tensorboard is run.
     required:
       - port
+  supervisor:
+    type: object
+    properties:
+      hostname:
+        type: string
+        description: |
+          The hostname to use for the supervisor.
+      port:
+        type: string
+        description: |
+          The port to use for the supervisor.
+      eval_base_path:
+        type: string
+        description: |
+          The base path to store the evaluation results.
+    required:
+      - hostname
+      - port
+      - eval_base_path
 required:
```

Comment:
is `metadata_processor` not required? please confirm

Reply:
metadata_processor currently is not used at all, indeed. that's a relict from old days, currently not needed.